SPECS/python-pyroute2/python-pyroute2.spec

```diff
@@ -1,90 +1,60 @@
 %global srcname pyroute2
 
-Summary:        Pure Python netlink library
-Name:           python-%{srcname}
-Version:        0.6.5
-Release:        3%{?dist}
-License:        GPLv2+
-URL:            https://github.com/svinota/%{srcname}
-Source0:        https://pypi.io/packages/source/p/pyroute2/%{srcname}-%{version}.tar.gz
-Source1:        %{srcname}.core-%{version}.tar.gz
-Source2:        %{srcname}.nslink-%{version}.tar.gz
-Source3:        %{srcname}.nftables-%{version}.tar.gz
-Source4:        %{srcname}.ethtool-%{version}.tar.gz
-Source5:        %{srcname}.ipset-%{version}.tar.gz
-Source6:        %{srcname}.ipdb-%{version}.tar.gz
-Source7:        %{srcname}.ndb-%{version}.tar.gz
-BuildArch:      noarch
-
-%description
-PyRoute2 provides several levels of API to work with Netlink
-protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,
+%global _description \
+PyRoute2 provides several levels of API to work with Netlink\
+protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,\
 IPQ.
 
-%package -n python3-%{srcname}
-Summary:        %{summary}
-BuildRequires:  python3-devel
-BuildRequires:  python3-setuptools
- 
-%description -n python3-%{srcname}
-PyRoute2 provides several levels of API to work with Netlink
-protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,
-IPQ.
+Name: python-%{srcname}
+Version: 0.7.12
+Release: 1%{?dist}
+Vendor: Microsoft Corporation
+Distribution: Azure Linux
+Summary: Pure Python netlink library
+License: GPL-2.0-or-later OR Apache-2.0
+URL: https://github.com/svinota/%{srcname}
+
+BuildArch: noarch
+BuildRequires: python3-pip
+BuildRequires: python3-wheel
+Source0: %{url}/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+%description %{_description}
+
+
+%package -n python%{python3_pkgversion}-%{srcname}
+Summary: %{summary}
+BuildRequires: python%{python3_pkgversion}-devel
+BuildRequires: python%{python3_pkgversion}-setuptools
+
+%description -n python%{python3_pkgversion}-%{srcname} %{_description}
 
 %prep
-%setup -q -n %{srcname}-%{version}
-cd ..
-tar xzvf %{SOURCE1}
-tar xzvf %{SOURCE2}
-tar xzvf %{SOURCE3}
-tar xzvf %{SOURCE4}
-tar xzvf %{SOURCE5}
-tar xzvf %{SOURCE6}
-tar xzvf %{SOURCE7}
+%autosetup -n %{srcname}-%{version}
+
+%generate_buildrequires
+%pyproject_buildrequires
```

Comment:
Since this doesn't work for us are we sure we have the right build time dependencies? I'm guessing the package builds, but I'm worried we may be missing some features disabled during build time due to missing Python modules.

Reply:
I needed to add the `BuildRequires` listed above because we don't have support for the dynamic dependencies. I will remove these lines to avoid confusion.